core-bundle/src/Messenger/MessageHandler/SearchIndexMessageHandler.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Messenger\MessageHandler;
+
+use Contao\CoreBundle\Messenger\Message\SearchIndexMessage;
+use Contao\CoreBundle\Search\Indexer\IndexerInterface;
+use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
+
+class SearchIndexMessageHandler implements MessageHandlerInterface
```

Comment:
```suggestion
use Symfony\Component\Messenger\Attribute\AsMessageHandler;

#[AsMessageHandler]
class SearchIndexMessageHandler
```

The interface seems to be deprecated: https://github.com/symfony/symfony/blob/6.2/UPGRADE-6.2.md#messenger

Reply:
Yes but I left it intentionally since we do not require 6.2 as minimum version.

Reply:
Ah but the attribute seems to be there since 5.4, so I can actually change it, yes.